src/examples/forms/multithumbrange/MultiThumbRangeHint.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { Field, Label, Hint, MultiThumbRange } from '@zendeskgarden/react-forms';
+
+const Example = () => {
+  const [minVal, setMinVal] = useState<number | undefined>(20);
+  const [maxVal, setMaxVal] = useState<number | undefined>(80);
+
+  return (
+    <Row justifyContent="center">
+      <Col sm={5}>
+        <Field>
+          <Label>Flowers</Label>
+          <Hint>
+            There are between {minVal} and {maxVal} flowers available
+          </Hint>
+          <MultiThumbRange
+            min={10}
+            max={150}
```

Comment:
Ah, nvmd. I actually misread the API. I thought there was a way to specify a min for the upper thumb and a max for the lower – which prevents having them meet in the middle and having hint text that reads something like "There are between 61 and 61 flowers available". Let's just call it an API shortcoming and remove these for now.